raiden/tests/integration/test_raidenservice.py

```diff
@@ -135,8 +120,8 @@ def num_matching_queued_messages(room: str, message_type: Type) -> int:
             )
 
         assert num_matching_queued_messages(MONITORING_BROADCASTING_ROOM, RequestMonitoring) == 1
-        assert num_matching_queued_messages(PATH_FINDING_BROADCASTING_ROOM, PFSFeeUpdate) == 1
-        assert num_matching_queued_messages(PATH_FINDING_BROADCASTING_ROOM, PFSCapacityUpdate) == 1
+        assert num_matching_queued_messages(PATH_FINDING_BROADCASTING_ROOM, PFSFeeUpdate) == 0
```

Comment:
I don't understand, why this PR would change the messages exchanged? And if it does, then the comment from lines `103-107` must be updated!